VEDA/settings.py

```diff
@@ -120,6 +129,7 @@
         'rest_framework.permissions.IsAuthenticated',
     ),
     'DEFAULT_AUTHENTICATION_CLASSES': (
+        'edx_rest_framework_extensions.authentication.JwtAuthentication',
```

Comment:
@nasthagiri can you please give it a look, purpose of this PR is to write an endpoint which is useable by platform [PR](https://github.com/edx/edx-platform/pull/16116/commits/5a2e063e24588b0114d4b87cb044f9f7e5c77071#diff-760a096297780408db8ee1ac7b3c9ad4R28)  , We're authenticating via JWT token, just wanted to confirm that this meets edX's standards.

Reply:
To minimize unnecessary exposure (per principle of least privilege), I agree that this class should be added to only the views that need JWT auth - so don't add it to the default list.

Reply:
Have default authentication classes is fine, and is a great way to ensure that your endpoints are always protected. I would keep this here, along with `SessionAuthentication` (useful testing in the browser) The others should go away given that none of them are used.

Reply:
@clintonb I am not sure if other authentication classes is used of not. @yro Do you know if there is any authentication classes that need to be removed from here that is not used anymore ?

Reply:
@mushtaqak The other classes are still in use by the workers.

Reply:
My rough guess is that this is being used to communicate with edx-val and video-worker but I am not sure.